scico/linop/_linop.py

```diff
@@ -58,6 +57,40 @@ def power_iteration(A: LinearOperator, maxiter: int = 100, key: Optional[PRNGKey
     return mu, v
 
 
+def operator_norm(A: LinearOperator, maxiter: int = 100, key: Optional[PRNGKey] = None):
```

Comment:
idea: it might be convenient to also have a norm property on `LinearOperator`s that can be set during construction if it is easy to compute. Circulant convolutions come to mind. Then optimizers that want the norm can see if `A.norm is not None` before running the potentially expensive power iteration.